cli/data/get.go

```diff
@@ -53,7 +53,7 @@ var getCmd = &cobra.Command{
 Example:
 
 $ paddle data get -b experimental --bucket roo-pipeline trained-model/version1 dest/path
-$ paddle data get -b experimental --bucket roo-pipeline --files file1.csv,file2.csv trained-model/version1 dest/path
+$ paddle data get -b experimental --bucket roo-pipeline --keys file1.csv,file2.csv trained-model/version1 dest/path
```

Comment:
I'm confused by the decision to name this `keys` instead of `files`. `files` seems much more user friendly and intuitive to me. Could you elaborate on why this decision was made?

Reply:
This decision came from this discussion here - https://github.com/deliveroo/paddle/pull/33#discussion_r293818056

Reply:
keys is the S3 sematics IMHO more correct here. In the future keys could also be `foo/bar/` and should download anything with that prefix